### DISCUSSION TITLE
如何去除登录权限控制和登录页面，我想使用这个做一个静态网站，不需要登录页面那些，打开就进入首页 应该咋操作呢

### DISCUSSION THREAD

**Discussion by 315574925** [Tencent/tdesign-vue-next-starter#752] [Q&A]
如何去除登录权限控制和登录页面，我想使用这个做一个静态网站，不需要登录页面那些，打开就进入首页 应该咋操作呢

**Comment by uyarn** [comment]
建议直接执行npx td-starter init 然后具体选择lite版本即可 就是一个轻量的初始化模板项目 不包括这些其他的页面和登录 权限控制等内容
